src/server/getBrowserTitle.js

```diff
@@ -0,0 +1,19 @@
+import config from 'config';
+
+export const TITLE_TEMPLATE = `%s | ${config.title}`;
+
+export default (location = {}, state = {}) => {
+  let titlePrefix;
+
+  const { pathname } = location;
+
+  // Petitions route
+  if (pathname && pathname.indexOf('/petitions') > -1) {
+    const { petition } = state;
+    titlePrefix = petition && petition.title;
+  }
+
+  return titlePrefix
+    ? TITLE_TEMPLATE.replace('%s', titlePrefix)
+    : config.title;
+};
```

Comment:
I am still a bit unsure of this file and how we proceed with more routes. It's concerned with getting the page title for each 'page type' and returning for the server-side markup (added in Nunjucks).

This happens before we've started the React render tree so we get the location of the request and the full state (passed from the store), then match the route with the title we want - here `petition.title`.